documentation/modules/upgrading/con-upgrade-listeners.adoc

```diff
@@ -0,0 +1,63 @@
+// This module is included in the following assemblies:
+//
+// upgrading/assembly_upgrade-kafka-versions.adoc
+
+[id='con-upgrade-listeners-{context}']
+= Updating listener configuration
+
+Strimzi provides a `GenericKafkaListener` schema for the configuration of Kafka listeners in a `Kafka` resource.
+
+`GenericKafkaListener` replaces the `KafkaListeners` schema, which is deprecated.
+
+The `GenericKafkaListener` schema is backwards compatible.
```

Comment:
I don't think this is really true. _The `listeners` field supports both the old deprecated form and the new array form._ would be a better way to put it.

Reply:
Changed and moved into the next paragraph:

> With the `GenericKafkaListener` schema, you can configure as many listeners as required,
> as long as their names and ports are unique.
> The listeners configuration is defined as an array.
> The `listeners` property supports the deprecated format and the array format.